library/Icingadb/Setup/RedisStep.php

```diff
@@ -133,6 +139,10 @@ public function getSummary()
                 new HtmlElement('strong', null, Text::create(t('Port'))),
                 $this->data['redis2_port'] ?: 6380
             ]));
+            $secondaryOptions->addHtml(Table::row([
+                new HtmlElement('strong', null, Text::create(t('Password'))),
+                $this->data['redis2_password'] ?: 6380
```

Comment:
Use `mt('icingadb', 'None', 'non-existence of a value')` as default here.

Reply:
changed this, too.